packages/fxa-settings/src/components/Settings/Tooltip/index.tsx

```diff
@@ -50,23 +50,23 @@ export const Tooltip = ({
       title={message}
       className={classNames(
         `z-50 absolute py-2 px-6 text-center text-white
-         p-3 rounded text-xs left-0 font-header font-bold
+         p-3 rounded text-xs font-header font-bold
           shadow-tooltip-grey-drop
          `,
         type === 'error' ? 'bg-red-600' : 'bg-grey-500',
         className,
         {
-          'left-1/2 transform -translate-x-1/2': !anchorLeft,
-          'left-0': anchorLeft,
+          'ltr:left-1/2 transform -translate-x-1/2': !anchorStart,
+          'ltr:left-0 rtl:right-0': anchorStart,
```

Comment:
Noice 👌 

I think `Tooltip` should be moved from `Settings/` to `/` since we're using it outside of Settings now.